projects/igniteui-angular/src/lib/combo/combo.component.html

```diff
@@ -33,7 +33,7 @@
                 (keyup)="handleKeyUp($event)" (keydown)="handleKeyDown($event)" (focus)="dropdown.onBlur($event)" [placeholder]="searchPlaceholder" aria-autocomplete="both"
                 [attr.aria-owns]="dropdown.id" [attr.aria-labelledby]="ariaLabelledBy" />
         </igx-input-group>
-        <ng-container *ngTemplateOutlet="headerTemplate; context: {$implicit: this}">
+        <ng-container *ngTemplateOutlet="headerTemplate ? headerTemplate : null ; context: {$implicit: this}">
```

Comment:
I assume the template outlet would handle the missing `headerTemplate` without this explicit check, right?